src/normalizer/chinese/kvariants.rs

```diff
@@ -0,0 +1,145 @@
+use once_cell::sync::Lazy;
+use serde::Deserialize;
+use std::collections::HashMap;
+use std::fs;
+
+#[derive(Debug, PartialEq)]
+pub enum KVariantClass {
+    Wrong,
+    SementicVariant,
+    Simplified,
+    Old,
+    Equal,
+}
+
+#[derive(Debug, PartialEq)]
+pub struct KVariant {
+    pub source_ideograph: char,
+    pub classification: KVariantClass,
+    pub destination_ideograph: char,
+}
+
+#[derive(Deserialize)]
+pub struct TsvRow {
+    lhs: String,
+    relation: String,
+    rhs: String,
+}
+
+pub static KVARIANTS: Lazy<HashMap<char, KVariant>> = Lazy::new(|| {
+    // The tab separated format is like:
+    //
+    //   㨲 (U+3A32)	wrong!	㩍 (U+3A4D)
+    //   铿 (U+94FF)	simp	鏗 (U+93D7)
+    //   㓻 (U+34FB)	sem	    剛 (U+525B)
+    //   ...
+    //
+    let file = fs::File::open("dictionaries/txt/chinese/kVariants.tsv").unwrap();
```

Comment:
Hi @ManyTheFish I didn't realize until now that I probably should have use `include_bytes!` instead of `File::open` here. Not sure if this will break when packaged and released. I'm working on a follow-up PR to fix it.

Reply:
Hello @choznerol, nice! Thank you!